### PR TITLE
Trev8939/update location data source samples

### PR DESCRIPTION
**Pull request by TADraeseke**
Just a small one to review when you have a minute. Thanks @hud10837!

### CODE REVIEW

File: java/navigate-route/src/main/java/com/esri/arcgisruntime/sample/navigateroute/MainActivity.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 package com.esri.arcgisruntime.sample.navigateroute;
 
 import java.util.ArrayList;
+import java.util.Calendar;
 import java.util.List;
 import java.util.concurrent.ExecutionException;
```
Comment (hud10837) [inline java/navigate-route/src/main/java/com/esri/arcgisruntime/sample/navigateroute/MainActivity.java:23]: Are these line breaks intentional?

File: java/navigate-route/src/main/java/com/esri/arcgisruntime/sample/navigateroute/MainActivity.java [hunk 2]
```diff
@@ -25,17 +26,19 @@
 import android.os.Build;
 import android.os.Bundle;
 import android.speech.tts.TextToSpeech;
-import androidx.appcompat.app.AppCompatActivity;
 import android.text.format.DateUtils;
 import android.util.Log;
 import android.widget.Button;
 import android.widget.TextView;
 import android.widget.Toast;
```
Comment (hud10837) [inline java/navigate-route/src/main/java/com/esri/arcgisruntime/sample/navigateroute/MainActivity.java:34]: see above

File: java/navigate-route/src/main/java/com/esri/arcgisruntime/sample/navigateroute/MainActivity.java (hunk 1 above)
Comment (TADraeseke) [reply java/navigate-route/src/main/java/com/esri/arcgisruntime/sample/navigateroute/MainActivity.java:23]: Oh yeah, we used to seperate API imports out. Not possible in Kotlin at the moment, so we're not doing it for kotlin samples.

**Comment by TADraeseke** [comment]
Ooo yeah. I forgot we had a Kotlin version of this sample. Can you re-review now that Kotlin has been updated too? Thanks @hud10837!

**Pull request merged**